app/interfaces/api/v1/external_users/claims/advocate_claim.rb

```diff
@@ -27,15 +27,15 @@ def local_t(attribute)
       namespace '/' do
         desc 'DEPRECATED: Create an Advocate final claim. see advocates/final endpoint'
         post do
-          deprecate(datetime: Time.new(2019, 8, 30), link: "#{request.base_url}/api/release_notes")
+          deprecate(datetime: Time.new(2020, 8, 30), link: "#{request.base_url}/api/release_notes")
```

Comment:
Why have these dates changed?

Reply:
`./spec/support/shared_examples_for_api.rb:325` expected the current time to be less than or equal to the above time.
As we are now in September, the spec expected time is now greater, therefore tests were failing.
I set a new time in the future in order to mock the test case and allow it to pass the given scenario.